pkg/asset/installconfig/aws/permissions.go

```diff
@@ -38,6 +38,9 @@ const (
 
 	// PermissionDeleteHostedZone is a set of permissions required when the installer destroys a route53 hosted zone.
 	PermissionDeleteHostedZone PermissionGroup = "delete-hosted-zone"
+
+	// PermissionKMSEncryptionKeys is an additional set of permissions required when the installer uses user provided kms encryption keys.
+	PermissionKMSEncryptionKeys PermissionGroup = "kms-encryption-keys"
```

Comment:
The permission group should cover a set of permissions. I don't see this one added to the permissions map below.

Reply:
The list of permissions is yet to be finalized.   https://issues.redhat.com/browse/OCPBUGS-13664?focusedId=22816098&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-22816098

Reply:
Added the permissions